src/SplitView.cpp

```diff
@@ -137,42 +160,56 @@ SplitView::SplitView(const char* name, const TransactionData& trans, const int32
 
 	BLayoutBuilder::Group<>(fSplitContainer, B_VERTICAL, 0)
 		.SetInsets(0)
+		.Add(new BSeparatorView(B_HORIZONTAL, B_PLAIN_BORDER))
+		.AddStrut(B_USE_SMALL_SPACING)
 		.AddGrid(1.0f, 0.0f)
-		.Add(fAddSplit, 0, 0)
-		.Add(fRemoveSplit, 1, 0)
-		.AddGlue(2, 0)
-		.Add(fSplitTotal, 3, 0)
-		.AddGlue(4, 0)
-		.End()
+			.Add(fAddSplit, 0, 0)
```

Comment:
if you want to keep this formatting, you'll have to add `// clang-format off  ` then `// clang-format on`

Reply:
Thanks for the reminder! PR was updated accordingly.